src/components/ErorrBanner.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+
+import styled from 'styled-components'
+
+const ErrorDiv = styled.div`
+  max-width: 400px;
+  background-color: ${p => p.theme.colors.error};
+  position: fixed;
+  bottom: ${p => (p.shown ? '10px' : '-100px')};
```

Comment:
is hiding it just shifting it up? should we also set opacity here?

Reply:
I don't think its really necessary since its position is fixed and outside the window.

Reply:
Actually I'll add it just in case there are some weird styling issues on smaller screens